This is an assignment on python exceptions
